tmp/cjs/react-jsx-dev-runtime.development.js

```diff
@@ -99,7 +99,16 @@ function printWarning(level, format, args) {
   // update consoleWithStackDev.www.js as well.
   {
     var ReactDebugCurrentFrame = ReactSharedInternals.ReactDebugCurrentFrame;
-    var stack = ReactDebugCurrentFrame.getStackAddendum();
+    var stack = '';
+
+    if (currentlyValidatingElement) {
```

Comment:
This is a manual change replicating what createElement does, but with a slightly different wiring.